tests/system/test_cluster/test_agent_files_deletion/test_agent_files_deletion.py

```diff
@@ -1,30 +1,32 @@
 # Copyright (C) 2015-2021, Wazuh Inc.
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
-
+import os
 from os.path import join, dirname, abspath
-from time import sleep
+from time import time, sleep
 
 import pytest
-
-from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.tools import WAZUH_PATH, WAZUH_LOGS_PATH
+from wazuh_testing.tools.monitoring import HostMonitor
 from wazuh_testing.tools.system import HostManager
 
 master_host = 'wazuh-master'
 worker_host = 'wazuh-worker2'
 agent_host = 'wazuh-agent3'
+local_path = os.path.dirname(os.path.abspath(__file__))
+messages_path = os.path.join(local_path, 'data/messages.yml')
+tmp_path = os.path.join(local_path, 'tmp')
 managers_hosts = [master_host, worker_host]
 inventory_path = join(dirname(dirname(dirname(abspath(__file__)))), 'provisioning', 'basic_cluster', 'inventory.yml')
 host_manager = HostManager(inventory_path)
-time_to_sync = 60
+time_to_sync = 10
+time_to_agent_reconnect = 180
 
 # Each file should exist in all hosts specified in 'hosts'.
 files = [{'path': join(WAZUH_PATH, 'queue', 'rids', '{id}'), 'hosts': managers_hosts},
          {'path': join(WAZUH_PATH, 'queue', 'agent-groups', '{id}'), 'hosts': managers_hosts},
          {'path': join(WAZUH_PATH, 'queue', 'diff', '{name}'), 'hosts': [worker_host]},
          {'path': join(WAZUH_PATH, 'queue', 'db', '{id}.db'), 'hosts': [worker_host]}]
-db_queries = ["select * from agent where id={id}",
-              "select * from belongs where id_agent={id}"]
```

Comment:
This use case is no longer being tested, please add it again so that the test makes sure that the `belongs` table is also updated.